snapshot.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: nightly-2020-12-14
+resolver: nightly-2021-01-05
```

Comment:
Note that this moves us from GHC 8.10.2 to 8.10.3. I don't expect any problems as a result of that, but it's something to be aware of.